src/overlays/actors/ovl_En_Cow/z_en_cow.h

```diff
@@ -3,21 +3,37 @@
 
 #include "ultra64.h"
 #include "global.h"
+#include "assets/objects/object_cow/object_cow.h"
+
+#define EN_COW_FLAG_PLAYER_HAS_APPROACHED (1 << 1)
+#define EN_COW_FLAG_WONT_GIVE_MILK (1 << 2)
+
+#define EN_COW_TYPE(thisx) ((thisx)->actor.params)
+typedef enum {
+    /*  0 */ EN_COW_TYPE_DEFAULT,
```

Comment:
Ah I was about to suggest "en_cow_type_cow" because "default" looks weird, but MM has `EN_COW_TYPE_ABDUCTED` so it makes sense there, hm

this kind of thing is hard to rule, since it's anachronistic to name things in OoT based on MM, but also the consistency is nice

I guess I'd say `EN_COW_TYPE_COW` would be better since "default" doesn't make sense in OoT, but I'm open to more opinions

Reply:
then I think COW is better, typically these enums are important to be understood simply when used externally, to avoid having to browse through the code again to figure them out